packages/lib/package.json

```diff
@@ -24,6 +24,7 @@
     "react": "^17.0.0"
   },
   "dependencies": {
+    "@leafygreen-ui/tokens": "^2.0.1",
```

Comment:
I don't love importing other LG packages from `lib`, since it can introduce circular dependencies, but this one seems fine

Reply:
I did this so we can define the `baseFontSize` storybookArgType as `options: Object.values(BaseFontSize)`, but I don't think it's worth the risk. I've duplicated the font size values here